python/ray/tests/test_actor.py

```diff
@@ -264,6 +264,24 @@ def get_values2(self, f3):
     assert results2[2].x == 3
 
 
+def test_actor_class_attributes(ray_start_regular):
```

Comment:
I think we have an existing `test_actor_class_inheritance` or similar where we should put this instead. Also, could you add a test for multiple levels of inheritance? e.g.,
```
class Grandparent(object):
    X = 3

class Parent(Grandparent):
    pass

@ray.remote
class Child(Parent):
    pass
```

Reply:
And one for multiple inheritance (`class Child(Parent1, Parent2)`).

Reply:
I added the test. I don't think it makes sense to add it to `test_actor_class_inheritance` because it has nothing to do with inheritance, besides the implementation. It's just about the actor keeping class attributes.